tests/api.c

```diff
@@ -40062,7 +40062,40 @@ static void test_wolfSSL_X509_get_ext_by_NID(void)
     AssertIntEQ(rc, -1);
 
     wolfSSL_X509_free(x509);
+#endif
+}
+
+static void test_wolfSSL_X509_get_ext_subj_alt_name(void)
+{
+#if defined(OPENSSL_ALL) && !defined(NO_RSA)
+    int rc;
+    FILE* f;
```

Comment:
I see a mix of linux BSK fopen and XFILE/XFOPEN in api.c, but my preference is to use our wc_port.h and XF macros.